hugo/assets/scss/components/tree.scss

```diff
@@ -187,4 +187,16 @@
             }
         }
     }
+
+    .is-book > :not(.is-active-path) {
+        .is-chapter {
+            .has-children {
+                margin: 0 0 6px;
+            }
+        }
+
+        .is-page {
+            display: none;
```

Comment:
Maybe it's better to not render them in hugo instead of hiding them.

Reply:
I've taken over the branch and moved the logic to hugo so this comment is obsolete now